### ISSUE TITLE
new api version

### ISSUE THREAD

**Issue by sethlyons** [sethlyons/duo_security#3]
do you plan to continue supporting this gem?  if so i'll submit a PR in the next week or so to support the new version (v2) of the duo api.
thanks.

**Comment by timraasveld** [comment]
Hey, thanks for your offer.

If you submit your PR, we can review it and give you push access if it looks good.

**Comment by sethlyons** [comment]
#4 submitted

**Comment by sethlyons** [comment]
also, if you guys are no longer planning to support this gem, i'd be happy to accept a transfer to my github org.

**Comment by sethlyons** [comment]
just wanted to follow up on the status of this PR.  thanks!

**Comment by timraasveld** [comment]
Sorry for the late reply.

I discussed the matter with my colleagues, and we decided that it's better if you maintain your own fork since we no longer use the gem ourselves and don't want to spend time reviewing PRs.

If you would like to be the new maintainer of the project, we could put up a link telling people to use your fork. What do you think?

**Comment by sethlyons** [comment]
sure, no problem.  do you prefer i maintain a new fork and new gem instead of transferring ownership of this repo and the existing gem to me?

**Comment by timraasveld** [comment]
Yes, that would be perfect. That means that if one day you turn evil and put dangerous code in the repo (not that I think that will happen :) ), it's not associated to our organization.

We could give you push rights for the gem. Do you want to use the e-mail address on your profile?

**Comment by sethlyons** [comment]
>>Yes, that would be perfect. That means that if one day you turn evil and put dangerous code in the repo (not that I think that will happen :) ), it's not associated to our organization.
Understood.

>>We could give you push rights for the gem. Do you want to use the e-mail address on your profile?
Yes, please.  Thanks!
